public/locales/pl/translation.json

```diff
@@ -39,5 +39,21 @@
     "contact": "Kontakt",
     "email": "myslaceklasy@email.com",
     "copyrights": "© Copyrights 2020."
+  },
+  "loginPage": {
+    "loginHeader": "Zaloguj się",
+    "loginSubtitle": "*Opcja logowania dostępna wyłącznie dla naszych Moderatorów.",
+    "loginErrorMessage": "Nieprawidłowy login lub hasło",
+    "emailErrorMessage": "Błędny email",
+    "passwordErrorMessage": "Błędne hasło",
```

Comment:
These descriptions are unused because we had problems with adding useTranslation inside `validations.tsx`